startup.d/20-syntax-check.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+python -m py_compile $(find . -name '*.py');
+if [ "$?" != "0" ]
+then
+  echo "There is syntax error. This may endanger chaos. Pauses updates."
+  killall bash
```

Comment:
1. Chaos runs, merges PRs, and sees it needs an update.
2. Chaos replaces itself with `startup.sh`. At this point, `chaos.py` is no longer running.
3. `startup.sh` runs, sees the new Python code doesn't compile, and kills itself.
4. Supervisor notices chaos died and attempts to restart it. The new code\* fails repeatedly.

\* I'll be honest, I haven't looked into Supervisor at all, but I would find it hard to believe that it keeps the old copy around until reread is issued. It seems much more likely to me that it will execute the newly pulled, bad code.